README.md

```diff
@@ -54,6 +54,8 @@ Delete Gemfile.lock and run `bundle install`
 bundle exec jekyll serve
 ```
 
+If you have a problem running the command above, add `vendor` at the section exclude in the _config.yml file.
```

Comment:
@RoboticMind what are your thoughts on this?

Reply:
Maybe should just add the config change to _config.yml instead?

Reply:
Done! I added `vendor` at the section exclude in _config.yml file and translated another page to portuguese

Reply:
this comment can be removed now, correct?

Reply:
Yes, now I removed. 😅